src/MCPClient/lib/clientScripts/move_to_backlog.py

```diff
@@ -13,6 +12,30 @@
 import storageService as storage_service
 
 
+class StorageServiceCreateFileError(Exception):
+    pass
+
+
+def _create_file(transfer_uuid, current_location, relative_transfer_path,
+                 backlog, backlog_path, size):
+    new_file = storage_service.create_file(
+        uuid=transfer_uuid,
+        origin_location=current_location['resource_uri'],
+        origin_path=relative_transfer_path,
+        current_location=backlog['resource_uri'],
+        current_path=backlog_path,
+        package_type='transfer',  # TODO use constant from storage service
+        size=size,
+    )
```

Comment:
`create_file` looks like it can raise `requests.exceptions.RequestException` and `storage_service.ResourceNotFound`. Maybe those should be caught here and re-raised as `StorageServiceCreateFileError`. Or maybe your `except Exception as err:` below should be `except (StorageServiceCreateFileError, storage_service.ResourceNotFound, requests.exceptions.RequestException) as err:`. Either way, the idea would be to be more explicit about which exceptions you expect to be catching.